packages/query-core/src/index.ts

```diff
@@ -11,6 +11,19 @@ export { MutationCache } from './mutationCache'
 export type { MutationCacheNotifyEvent } from './mutationCache'
 export { MutationObserver } from './mutationObserver'
 export { notifyManager, defaultScheduler } from './notifyManager'
+export {
+  managedSetTimeout,
+  managedClearTimeout,
+  managedSetInterval,
+  managedClearInterval,
+  systemSetTimeoutZero,
+  defaultTimeoutProvider,
+  timeoutManager,
```

Comment:
since the framework adapters re-export everything from the query-core, all of these now become part of the adapter APIs. I think it would be better if we just exported the `timeoutManager` and called the class methods directly, unless this has a large impact on bundle size.

Reply:
👍🏻